web-components/src/[sandbox]/examples/card.ts

```diff
@@ -1,11 +1,35 @@
 import "@/components/card/Card";
 import { html } from "lit-element";
 
+const cardMenu = ["Edit", "View", "Duplicate", "Delete"];
+
 export const cardTemplate = html`
   <h3>Default Card</h3>
-  <md-card>
+  <md-card
+    .menuOption=${cardMenu}
+    id="123456789"
+    title="Team A Report - Q1"
+    subtitle="Updated 2 hours ago"
+    fullscreen
+    @card-click=${(e: MouseEvent) => {
```

Comment:
can you show the even handling here? e.g. how to handle "Edit" click vs other menu items being clicked
Also the card itself needs to be clickable - and the menu items should emit a different custom event when clicked

Reply:
Done